CMakeLists.txt

```diff
@@ -89,7 +89,10 @@ add_project_dependency(
   "eigen3 >= 3.0.5"
 )
 if(BUILD_WITH_VECTORIZATION_SUPPORT)
-  add_project_dependency(Simde REQUIRED)
+  add_project_dependency(Simde)
+  if(NOT Simde_FOUND)
+    message(FATAL_ERROR "Set BUILD_WITH_VECTORIZATION_SUPPORT=OFF or install Simde.")
```

Comment:
I would rather suggest defining a hint variable in the FindSimde.cmake file and fill it in the CMakeLists.txt.
In this way, we can get the benefit of both worlds